tests/subsys/dfu/fmfu_fdev/src/main.c

```diff
@@ -52,11 +53,22 @@ uint8_t mfu_pk[] = {
 	0xdb, 0x04, 0x6c, 0x8a, 0x71, 0x87, 0x08, 0xda
 };
 
-static void test_mfu_ext_load_segments(void)
+
+static void test_real(void)
+{
+	COSE_Sign1_Manifest_t manifest;
+	size_t len;
+
+	bool res = cbor_decode_Wrapper(real_pk, sizeof(real_pk), &manifest,
+				       &len);
+	zassert_true(res, "FAIL");
+}
+
+static void test_fmfu_fdev_load_segments(void)
```

Comment:
I suggest removing `test_fmfu_fdev_load_segments` and `test_fmfu_fdev_prevalidate` and adding a test for `fmfu_fdev_load` that copies a blob from one place to another in flash.

Reply:
I agree. Speaking from experience, it is smart to wait a bit with the full fledged tests to later in the review process, as things are bound to change, perhaps a lot :)

Reply:
Made a task.